core/store/migrations/migration1592355365/migrate.go

```diff
@@ -0,0 +1,24 @@
+package migration1592355365
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate ensures that heads are unique and adds parent hash for use in reorg detection
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		CREATE TABLE "job_spec_errors" (
+			"id" bigserial primary key,
+			"job_spec_id" uuid REFERENCES job_specs(id) ON DELETE CASCADE NOT NULL,
+			"description" text NOT NULL,
+			"occurences" integer DEFAULT 1 NOT NULL,
+			"created_at" timestamptz NOT NULL,
+			"updated_at" timestamptz NOT NULL
+		);
+
+		CREATE UNIQUE INDEX job_spec_errors_unique_idx ON job_spec_errors ("job_spec_id", "description");
+		CREATE INDEX job_spec_errors_created_at_idx ON job_spec_errors USING brin (created_at);
+		CREATE INDEX job_spec_errors_updated_at_idx ON job_spec_errors USING brin (updated_at);
+		CREATE INDEX job_spec_errors_occurences_idx ON job_spec_errors (occurences);
```

Comment:
I believe you meant to spell it `occurrence`

Third time's the charm ;)

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: :man_facepalming: my grade school teachers are so disappointed rn

Reply:
Grep included 52 identical misspellings in yarn packages. At least I'm not the only one out there :joy:.